src/ui/components/Notifications/Notifications.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback } from 'react';
+import axios from 'axios';
+import { useQueryClient } from 'react-query';
+
+import PropTypes from 'prop-types';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import HistoryIcon from '@material-ui/icons/History';
+import DoneIcon from '@material-ui/icons/Done';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import useStyles from './styles';
+
+const Notifications = props => {
+  const classes = useStyles();
+  const queryClient = useQueryClient();
+
+  const { notifs } = props;
+  const updateNotif = useCallback(notif => {
+    return () => {
+      notif.viewed = true;
+      axios.put(`/collection/notifications?id=${notif.id}`, notif).then(() => {
+        queryClient.invalidateQueries('notifications');
+      });
+    };
+  });
+
+  const getElapsed = time => {
```

Comment:
In the Pathway Builder we used the [moment](https://www.npmjs.com/package/moment) library. The README says it is now a legacy project but I would use a library here instead of doing this manually